### ISSUE TITLE
Missing alt text on Search button

### ISSUE THREAD

**Issue by tomsedory** [OpenEugene/little-help-book-web#175]
The Search button in the header is missing an alt text tag. See this screen shot from the [WAVE extension](https://wave.webaim.org/extension/) that evaluates http sources for [WCAG compliance](https://www.w3.org/WAI/standards-guidelines/wcag/).

![image](https://user-images.githubusercontent.com/75692630/104684852-b8d1fd80-56ae-11eb-92fd-1681847336e4.png)

Note: Missing alt text is one of the top accessibility issues for blind users using screen readers.
